src/centralserver/admin-service/application/src/intTest/resources/behavior/api/timestampingServicesApi.feature

```diff
@@ -13,6 +14,7 @@ Feature: Timestamping services API
   @Modifying
   Scenario: Deleting timestamping service
     Given timestamping service is added
+    And timestamping services returns added timestamping service by id
```

Comment:
I would suggest to have a separate scenario for retrieving the timestamping service by id. It is easier to maintain and to do modifications in the future if needed.